app/presenters/project_presenter.rb

```diff
@@ -37,9 +40,9 @@ def block
   end
 
   # enumerates pairs of stages, the corresponding comparison object, and severity score
-  def compared_stages
+  def compared_stages # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Surprised about this too but 🤷

Reply:
Personally I'd always disable `Metrics/CyclomaticComplexity`. It leads to unnecessary refactoring and code massaging for the sake of getting a good rate.